scanners/port_scanner.py

```diff
@@ -0,0 +1,25 @@
+import nmap
+import os
+import time
+
+
+class PortScanner():
+
+    def __init__(self):
+        # This is needed for nmap static library to be added to the path
+        os.environ['PATH'] = os.environ['PATH'] \
+            + ':' + os.environ['LAMBDA_TASK_ROOT'] \
+            + '/bin/nmap-standalone/'
```

Comment:
To me, this is a bit weird to see something that globally affects the run-time buried inside a class instantiation.

I wonder if setting it at run-time of scanTCP and unsetting it after is more appropriate so we're not permanently affecting the run-time env.

I guess I don't exactly have a good feeling about where this should live yet.

Reply:
Fair point, and you are probably right, as if we package other binaries that perform other scans, the same line would have to present in other scanner classes too, which is a bad implementation.
Now that I'm thinking about this, we can probably make this assignment in `serverless.yml`. I will move it there and see nmap scanning still works.

Reply:
It does not appear possible to specify this in `serverless.yml` file. When I tried, it expects the same env variables to be present on my local system, and takes their values if present (rather than the env variables of the lambda environment). This is not what we want.

The official docs appear to recommend setting this at the top of the handler code/function. Setting it here and the unsetting in the end is probably the best option.

Another option is to hardcode the `PATH` variable in serverless.yml for only the functions that run TCP scans.

Reply:
I moved this to under the relevant handler functions...Still probably not the best place, but not sure if we have much choice yet.

We can revisit after simplifying the handler code.